src/components/onboardingGuide/OnboardingGuide.tsx

```diff
@@ -65,66 +66,47 @@ function OnboardingGuide({
-                            <a
-                                className="dc__link-n9 cn-9"
-                                href={PREVIEW_DEVTRON}
-                                rel="noreferrer noopener"
-                                target="_blank"
-                                onClick={onClickPreviewCard}
-                            >
-                                <img className="onboarding-card__img dc__top-radius-4" src={PreviewImage} />
-                                <div className="fw-6 fs-16 pt-32 pb-32 pl-24 pr-24 dc__break-word">
-                                    Explore a preconfigured Demo app at <span className="cb-5">preview.devtron.ai</span>
-                                </div>
-                            </a>
-                        </div>
-                        <div className="onboarding__line" />
-                        <div
-                            className="onboarding-card bcn-0 w-300 br-4 en-2 bw-1 cursor"
-                            onClick={onClickedDeployManageCardClicked}
-                        >
-                            <NavLink
-                                to={`${match.path}/${URLS.GUIDE}`}
-                                className="dc__no-decor fw-6 cursor cn-9"
-                                activeClassName="active"
-                            >
-                                <img
-                                    className="onboarding-card__img dc__top-radius-4"
-                                    src={HelmCollage}
-                                    alt="Deploy and manage helm"
-                                />
-                                <div className="fw-6 fs-16 pt-32 pb-32 pl-24 pr-24 dc__break-word">
-                                    Deploy and manage helm applications
-                                </div>
-                            </NavLink>
-                        </div>
-
-                        <div className="onboarding-card bcn-0 w-300 br-4 en-2 bw-1 cursor">
-                            <NavLink
-                                to={redirectDeployCardToCICD()}
-                                className="dc__no-decor fw-6 cursor cn-9"
-                                activeClassName="active"
-                                onClick={onClickedCICD}
-                            >
-                                <img
-                                    className="onboarding-card__img dc__top-radius-4"
-                                    src={DeployCICD}
-                                    alt="Please connect cluster"
-                                />
-                                <div className="fw-6 fs-16 pt-32 pb-32 pl-24 pr-24">
-                                    Deploy custom applications using CI/CD pipelines
-                                </div>
-                            </NavLink>
-                        </div>
+                        <ContentCard
```

Comment:
Could` we rename it to more specific like `OnboardingContentCard`

Reply:
@shivani170 It's not specific to Onboarding. This component is common & can be used anywhere which needs content to be shown in the same design.

Reply:
Done